src/napari_imagej/java.py

```diff
@@ -730,3 +730,25 @@ def LabelingIOService(self):
 
 
 jc = JavaClasses()
+
+
+@JImplementationFor("org.scijava.module.ModuleItem")
+class ModuleItemAddons(object):
+    """ModuleItem adons.
+
+    This class should not be initialized manually. Upon initialization
+    the ImagePlusAddons class automatically extends the Java
+    ij.ImagePlus via JPype's class customization mechanism:
+
+    https://jpype.readthedocs.io/en/latest/userguide.html#class-customizers
+    """
+
+    # RESTRICTED_NAMES = {
+    #     "in": "input",
+    # }
+
+    def py_name(self):
+        name = self.getName()
```

Comment:
`String`s are always automatically converted, right? Should we throw in an `ij().py.from_java()` here, just to be sure? Then we could remove the `ij().py.from_java` call in `_module_utils`.

Reply:
Fixed!